app/templates/views/roadmap.html

```diff
@@ -9,29 +9,46 @@
 
   <h1 class="heading-large">Roadmap</h1>
 
-  <p class="govuk-body">The GOV.UK Notify roadmap shows the things we’re working on and when we hope to have them ready for you to use.</p>
-  <p class="govuk-body">The roadmap is a guide to what we have planned, but some things might change.</p>
+  <p class="govuk-body">The GOV.UK Notify roadmap shows what we’re working on and some of the <a class="govuk-link govuk-link--no-visited-state" href="#things-we-have-done">things we’ve done</a>.</p>
+  <p class="govuk-body">The roadmap is only a guide. It does not cover everything we do, and some things may change.</p>
+  <p class="govuk-body">We review this page every 3 months. It was last updated on 24 November 2021.</p>
   <p class="govuk-body">You can <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('.support')}}">contact us</a> if you have any questions about the roadmap or suggestions for new features.</p>
 
-  <h2 class="heading-medium">July to December 2021</h2>
+  <h2 class="heading-medium">Things we’re working on</h2>
+
+<h3 class="heading-small">Now</h3>
 
   <ul class="list list-bullet">
     
-    <li>Help teams to write better messages (private alpha)</li>
-    <li>Investigate additional letter formats</li>
-    <li>Start sending emails from nhs.uk, gov.scot and parliament.uk email addresses</li>
+    <li>Meet our targets for <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('.accessibility-statement')}}">accessibility</a></li>
```

Comment:
```suggestion
    <li>Meet our targets for <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('.accessibility_statement')}}">accessibility</a></li>
```

This needs to be based on the function name, rather than the URL path. It's currently causing the tests to fail as Flask can't resolve the route:

```
Could not build url for endpoint 'main.accessibility-statement'. Did you mean 'main.accessibility_statement' instead?
```

Reply:
Thank you @benthorner